xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -46,14 +46,13 @@ CGUIWindowPVRGuide::CGUIWindowPVRGuide(bool bRadio) :
   m_refreshTimelineItemsThread(new CPVRRefreshTimelineItemsThread(this)),
   m_bRefreshTimelineItems(false)
 {
-  m_cachedTimeline = new CFileItemList;
+  m_cachedTimeline = std::make_shared<CFileItemList>();
```

Comment:
Just use: std::shared_ptr<CFileItemList> p2(new CFileItemList());

the only think that one looses is a bit of expeception saftiness and perhaps a little overhead

@MaxKellermann knows most likely